packages/pretty-format/src/plugins/AsymmetricMatcher.ts

```diff
@@ -80,6 +81,12 @@ export const serialize: NewPlugin['serialize'] = (
     );
   }
 
+  if (!isA('Function', val.toAsymmetricMatcher)) {
+    throw new Error(
+      `Asymmetric matcher ${val.constructor.name} does not implement toAsymmetricMatcher()`,
```

Comment:
Great! Could you add a test for this as well?

Reply:
Done